nixos/platform/network.nix

```diff
@@ -367,23 +368,17 @@ in
             }
           )) bridgedInterfaces) ++
         (lib.optionals (!isNull fclib.underlay)
+          # loopback dummy device
           ([(lib.nameValuePair
-            "fc-lldp-to-altnames"
+            "ul-loopback-netdev"
```

Comment:
I'm assuming the naming convention here and the next onw (x-netdev and network-addresses-x) are to conform with upstream naming schema?

Reply:
Yes, precisely. I've noted this in the commit message in 8878dd2.